rails-install-ubuntu.sh

```diff
@@ -2,50 +2,69 @@
 
 set -e
 
+echo "Adding PPA for up-to-date Node.js runtime. Give your password when asked."
+sudo add-apt-repository ppa:chris-lea/node.js
+
+echo "Adding deb. repository for PostgreSQL"
```

Comment:
The Postgres setup below worked, however as with the Node.js repository, the default Ubuntu packages are outdated by usually several months, if not more, since they have to go through a long application and testing process. The deb. repositories and PPAs solve this by being able to download packages right from the people who actually make them.